package com.divya.inventory_service.repository;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;
import com.divya.inventory_service.entity.Inventory;

public interface InventoryRepository
        extends JpaRepository<
                Inventory,
                Long> {

    Optional<Inventory>
    findByProductId(
            Long productId);
}